RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETLinkedType.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace RepositoryCompiler.CodeModel.CaDETModel.CodeItems
+{
+    public class CaDETLinkedType
```

Comment:
A summary comment would be helpful here. It should contain a brief description of what the linkedType is and an example or two.